app/assets/stylesheets/custom.scss

```diff
@@ -153,6 +153,10 @@ td {
   @include box-sizing;
 }
 
+#complete_your_loan {
```

Comment:
Avoid using id selectors<br>Selector `complete_your_loan` should be written in lowercase with hyphens